src/clusterfuzz/_internal/tests/appengine/handlers/cron/schedule_corpus_pruning_test.py

```diff
@@ -18,7 +18,7 @@
 from clusterfuzz._internal.datastore import data_types
 from clusterfuzz._internal.tests.test_libs import helpers
 from clusterfuzz._internal.tests.test_libs import test_utils
-from handlers.cron import schedule_corpus_pruning
+from clusterfuzz._internal.cron import schedule_corpus_pruning
```

Comment:
Per https://github.com/google/clusterfuzz/actions/runs/6116552082/job/16601925439?pr=3322

The ordering of the new import is wrong. the .cron needs to come first in this import block because imports need to be lexicographically ordered.